spark/ingestion/src/main/scala/feast/ingestion/BasePipeline.scala

```diff
@@ -33,11 +33,17 @@ object BasePipeline {
     val conf = new SparkConf()
 
     jobConfig.store match {
-      case RedisConfig(host, port, ssl) =>
+      case RedisConfig(host, port, auth, ssl) if auth.isEmpty =>
         conf
           .set("spark.redis.host", host)
           .set("spark.redis.port", port.toString)
           .set("spark.redis.ssl", ssl.toString)
+      case RedisConfig(host, port, auth, ssl) if auth.nonEmpty =>
+        conf
+          .set("spark.redis.host", host)
+          .set("spark.redis.port", port.toString)
+          .set("spark.redis.auth", auth)
```

Comment:
Shall we call it `spark.redis.password` ?

Reply:
That would not be possible because spark-redis package uses spark.redis.auth for the password field. We can modify this after we replace spark redis package with JedisCluster, however.